drizzle-orm/src/primary-key.ts

```diff
@@ -9,3 +9,9 @@ export abstract class PrimaryKey {
 
 	constructor(readonly table: Table, readonly columns: AnyColumn[]) {}
 }
+
+export abstract class PrimaryKeyBuilder {
```

Comment:
I'm not sure if this was the best approach based on other code in the repo.

The problem was that extractTablesRelationConfig was using `is(configEntry, PrimaryKeyBuilder)` with the Pg PrimaryKeyBuilder. 

I considered updating all the instances to have the same `entityKind` but I didn't see a lot of other cases of multiple classes sharking en entity kind, and suspect that is not the intended use case there.